kids-chat-app/src/components/chat/ChatInput.tsx

```
@@ -0,0 +1,79 @@
+    <div className="border-t-2 border-primary-100 bg-white p-4">
+      {/* Quick messages */}
+      <div className="flex gap-2 mb-3 overflow-x-auto pb-2">
+        {quickMessages.map((quick, index) => (
+          <button
+            key={index}
+            onClick={() => handleQuickMessage(quick.text)}
+            className="flex items-center gap-1 px-3 py-1 bg-secondary-100 hover:bg-secondary-200 rounded-full transition-colors whitespace-nowrap text-sm"
+            disabled={isLoading}
+          >
+            <span>{quick.emoji}</span>
+            <span>{quick.text}</span>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)